core-contracts/Dex/src/test/java/network/balanced/score/core/dex/DexImplTest.java

```diff
@@ -0,0 +1,343 @@
+package network.balanced.score.core.dex;
+
+import com.iconloop.score.test.Account;
+import com.iconloop.score.test.Score;
+import com.iconloop.score.test.ServiceManager;
+import com.iconloop.score.test.TestBase;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.any;
+
+
+import score.Address;
+import score.Context;
+
+import java.math.BigInteger;
+import java.util.Map;
+import java.util.HashMap;
+
+import static network.balanced.score.lib.test.UnitTest.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class DexImplTest extends TestBase {
```

Comment:
Tests looking really well structured so far :+1: 

```suggestion
import network.balanced.score.lib.test.UnitTest;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertTrue;


public class DexImplTest extends UnitTest {
```